resources/views/components/user/edit_form.blade.php

```diff
@@ -0,0 +1,63 @@
+@extends('components/layout')
+
+@section('content')
+    <div class="content-wrapper" id="formAdd">
```

Comment:
```suggestion
    <div class="content-wrapper" id="form-add">
```
id, class viết ntn nhé